source/IntegrationTests/Behaviours/IncomingRequests/GivenForwardMeteredDataV2Tests.cs

```diff
@@ -131,7 +131,7 @@ await GivenReceivedMeteredDataForMeteringPoint(
 
         // Arrange
         var requestMeteredDataForMeteringPointInputV1 = message.ParseInput<ForwardMeteredDataInputV1>();
-        var requestMeteredDataForMeteringPointAcceptedServiceBusMessage = MeteredDataForMeteringPointEventBuilder
+        var requestMeteredDataForMeteringPointAcceptedServiceBusMessage = ForwardMeteredDataResponseBuilder
```

Comment:
```suggestion
        var forwardMeteredDataAcceptedServiceBusMessage = ForwardMeteredDataResponseBuilder
```

Same goes for other places where the rename has happened